src/main/java/org/jenkinsci/plugins/ansible_tower/AnsibleTowerRunner.java

```diff
@@ -97,6 +100,7 @@ public boolean runJobTemplate(
         String expandedSkipJobTags = envVars.expand(skipJobTags);
         String expandedInventory = envVars.expand(inventory);
         String expandedCredential = envVars.expand(credential);
+        //String expandedScmBranch = envVars.expand(scmBranch);
```

Comment:
Is there a reason you don't want to expand variables found in the SCM Branch field?

Reply:
Fixed. It remains after my tests.